stable/consul/templates/consul.yaml

```diff
@@ -157,17 +182,11 @@ spec:
       - name: gossip-key
         secret:
           secretName: gossip-key
-      readinessProbe:
-        httpGet:
-          path: /v1/health/service/consul
-          port: {{.Values.HttpPort}}
-        initialDelaySeconds: 10
-        timeoutSeconds: 1
   volumeClaimTemplates:
   - metadata:
       name: datadir
       annotations:
-        volume.alpha.kubernetes.io/storage-class: anything
+        volume.beta.kubernetes.io/storage-class: "{{.Values.StorageClass}}"
```

Comment:
I've completed this although I'm not sure if it should be here. Reasoning is that StatefulSet is 1.5 and the corresponding storage annotation is now beta. 1.4 used PetSet and it used the storage used alpha notation. Thoughts?